website/content/en/status/report-2025-10-2025-12/openvox.adoc

```diff
@@ -0,0 +1,36 @@
+=== OpenVox (Puppet)
+
+Links: +
+link:https://voxpupuli.org/[Vox Pupuli] URL: link:https://voxpupuli.org/[] +
+link:https://github.com/OpenVoxProject/[OpenVox GitHub organization] URL: link:https://github.com/OpenVoxProject/[] +
+link:https://github.com/voxpupuli/[Vox Pupuli GitHub organization] URL: link:https://github.com/voxpupuli/[]
+
+Contact: Puppet Team <puppet@FreeBSD.org>
```

Comment:
If you have not done it already, I suggest you ask postmaster@ to create a new openvox@ address for you and to have puppet@ forwarded to openvox@. Then, please replace this line with
```suggestion
Contact: OpenVox Team <openvox@FreeBSD.org>
```